dataflows/car-processing/README.MD

```diff
@@ -317,7 +372,6 @@ Note that your connectors will continue to run in the background until you shut
 
 ```bash
 cdk deploy shutdown --name car-connector
-cdk deploy shutdown --name license-connector
```

Comment:
perhaps the line above should now read "Note that your connector will continue to run in the background until you shut it down"